packages/compose-cli/src/types.ts

```diff
@@ -10,7 +10,10 @@ export interface MeshComposeCLIConfig extends ComposeSubgraphsOptions {
    */
   output?: string;
   subgraphs: MeshComposeCLISubgraphConfig[];
+  /**
+   * TODO: Implement root level transforms later
   transforms?: MeshComposeCLITransformConfig[];
+   */
```

Comment:
I could not find the "root level transforms" on the [project roadmap](https://github.com/ardatan/graphql-mesh/issues/5357).

Any idea if it will?

Also, do you confirm this would be the way to implement custom directives?